alert/index.js

```diff
@@ -0,0 +1,58 @@
+var stompClient = null;
+var id = null;
+
+$(document).ready(function(){
+    connect();
+
+    var urlParams = new URLSearchParams(window.location.search);
+    id = urlParams.get('id');
+});
+
+function connect() {
+  var socket = new SockJS('https://localhost:8070/alert-websocket');
+  stompClient = Stomp.over(socket);
+    stompClient.connect({}, function (frame) {
+        console.log('Connected: ' + frame);
+        stompClient.subscribe('/alert/' + id, receivedAlert);
+    });
+}
+
+function disconnect() {
+    if (stompClient !== null) {
+        stompClient.disconnect();
+    }
+    setConnected(false);
+    console.log("Disconnected");
+}
+
+// testing code
+function sendAlert() {
```

Comment:
Is this the best spot to putting testing code?

Reply:
This function will be deleted.